.npmignore

```diff
@@ -1,6 +1,7 @@
 # Exclude everything but the contents of the dist directory.
 **/*
 !dist/**
+dist/video-js-*.zip
```

Comment:
Should it maybe just be `dist/**/*.zip`? That should capture all ZIP files in `dist` and its sub-directories (I think).

Reply:
I made it so it is same as in build script here:
https://github.com/videojs/video.js/blob/0b2a8e04ef704b363f63bec523e091ebe7101cb7/build/grunt.js#L196

Reply:
But zip files will probably never be needed in npm package. So ignoring all of them should be fine too.

Reply:
Either way works, I think. Did you want to make the update to ignore all zip files? Otherwise, this is good to go.